src/wp-includes/rest-api/endpoints/class-wp-rest-menu-items-controller.php

```diff
@@ -101,6 +101,22 @@ protected function check_has_read_only_access( $request ) {
 		);
 	}
 
+	/**
+	 * Retrieves a collection of menu items.
+	 *
+	 * @since 6.x.x
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 *
```

Comment:
```suggestion
```

Nit: No empty line between `@param` and `@return` in the WP docblock convention. Sad Clown.

🔢 This applies in multiple places.